app/_src/kubernetes-ingress-controller/guides/configuring-health-checks.md

```diff
@@ -45,7 +45,7 @@ For the purpose of this guide, we will setup an [httpbin](https://httpbin.org)
 service in the cluster and proxy it.
 
 ```bash
-$ kubectl apply -f https://bit.ly/k8s-httpbin
+$ kubectl apply -f https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/{{site.data.kong_latest_KIC.version}}/deploy/manifests/httpbin.yaml
```

Comment:
This will evaluate to `https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/2.8.1/deploy/manifests/httpbin.yaml` which doesn't exist. We should use `v{{site.data.kong_latest_KIC.version}}` instead as that's the pattern KIC uses for tags. 



The comment applies to all links in this PR.